DQMOffline/Trigger/python/BPHMonitor_cfi.py

```diff
@@ -30,24 +33,64 @@
   xmax  = cms.double(15),
 )
 
+hltBPHmonitoring.histoPSet.dRPSet = cms.PSet(
+  nbins = cms.int32 (  20 ),
+  xmin  = cms.double(   0),
+  xmax  = cms.double(20),
+)
+
+hltBPHmonitoring.histoPSet.massPSet = cms.PSet(
+  nbins = cms.int32 ( 200 ),
+  xmin  = cms.double(  2),
+  xmax  = cms.double(4.),
+)
+hltBPHmonitoring.histoPSet.dcaPSet = cms.PSet(
+  nbins = cms.int32 ( 40 ),
+  xmin  = cms.double(  0),
+  xmax  = cms.double(4.),
+)
+
+hltBPHmonitoring.histoPSet.dsPSet = cms.PSet(
+  nbins = cms.int32 ( 10),
+  xmin  = cms.double(  0),
+  xmax  = cms.double(100),
+)
+
+hltBPHmonitoring.histoPSet.cosPSet = cms.PSet(
+  nbins = cms.int32 ( 20),
+  xmin  = cms.double(  -1),
+  xmax  = cms.double(1),
+)
+hltBPHmonitoring.histoPSet.probPSet = cms.PSet(
+  nbins = cms.int32 ( 10),
+  xmin  = cms.double(  0),
+  xmax  = cms.double(1),
+)
+
 hltBPHmonitoring.tracks       = cms.InputTag("generalTracks") # tracks??
 hltBPHmonitoring.offlinePVs      = cms.InputTag("offlinePrimaryVertices") # PVs
 hltBPHmonitoring.beamSpot = cms.InputTag("offlineBeamSpot") # 
 hltBPHmonitoring.muons     = cms.InputTag("muons") # 
+hltBPHmonitoring.photons     = cms.InputTag("photons") # 
 
 hltBPHmonitoring.numGenericTriggerEventPSet.andOr         = cms.bool( False )
 hltBPHmonitoring.numGenericTriggerEventPSet.dbLabel       = cms.string("BPHDQMTrigger") # it does not exist yet, we should consider the possibility of using the DB, but as it is now it will need a label per path !
 hltBPHmonitoring.numGenericTriggerEventPSet.andOrHlt      = cms.bool(True)# True:=OR; False:=AND
+hltBPHmonitoring.numGenericTriggerEventPSet.andOrL1      = cms.bool(True)# True:=OR; False:=AND
 hltBPHmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults::HLT" )
-hltBPHmonitoring.numGenericTriggerEventPSet.hltPaths      = cms.vstring("HLT_Dimuon10_Jpsi_Barrel") # HLT_ZeroBias_v*
+hltBPHmonitoring.numGenericTriggerEventPSet.hltPaths      = cms.vstring("HLT_Mu7p5_TkX_Jpsi") # HLT_ZeroBias_v*
+hltBPHmonitoring.numGenericTriggerEventPSet.l1Algorithms      = cms.vstring("DM0_SQ") # HLT_ZeroBias_v*
 hltBPHmonitoring.numGenericTriggerEventPSet.hltDBKey      = cms.string("diMu10")
 hltBPHmonitoring.numGenericTriggerEventPSet.errorReplyHlt = cms.bool( False )
+hltBPHmonitoring.numGenericTriggerEventPSet.errorReplyL1 = cms.bool( True )
+hltBPHmonitoring.numGenericTriggerEventPSet.l1BeforeMask = cms.bool( True )
 hltBPHmonitoring.numGenericTriggerEventPSet.verbosityLevel = cms.uint32(0)
 
 hltBPHmonitoring.denGenericTriggerEventPSet.andOr         = cms.bool( False )
 hltBPHmonitoring.denGenericTriggerEventPSet.dcsInputTag   = cms.InputTag( "scalersRawToDigi" )
 hltBPHmonitoring.denGenericTriggerEventPSet.hltPaths  = cms.vstring( "HLT_Dimuon6_Jpsi_NoVertexing" )#reference
```

Comment:
Yes, that is right. And this is the reason why it does not matter what paths are listed in this file as far as I understand, because I change them in DQMOffline/Trigger/python/BPHMonitor_cff.py file anyway.